docs/configure/css-troubleshooting/angular.mdx

```diff
@@ -39,4 +39,28 @@ Additionally, if you need Storybook specific styles that are separate from your
     }
   }
 }
-```
\ No newline at end of file
+```
+
+### Nx 13 with Angular 13
+In the library which you (currently) use for storybook demonstrations (or for development with storybook), e.g. `non-prod-storybook` in `libs/non-prod/storybook/` you add the additional styles e.g. the styles from your default app `my-default-app` in the `project.json` like this:
+
+```json
+  "build-storybook": {
+    "executor": "@nrwl/storybook:build",
+    "outputs": ["{options.outputPath}"],
+    "options": {
+      "uiFramework": "@storybook/angular",
+      "outputPath": "dist/storybook/non-prod-storybook",
+      "config": {
+        "configFolder": "libs/non-prod/storybook/.storybook"
+      },
+      "projectBuildConfig": "non-prod-storybook:build-storybook",
+      "styles": ["apps/my-default-app/src/styles.scss"]
+    },
+    "configurations": {
+      "ci": {
+        "quiet": true
+      }
+    }
+  }  
+```
```

Comment:
@juri-sinitson based on your pull request we have a bit of room for improvement. We can probably go with:
```md

### Nx with Angular 13

If you're working with Storybook and [NX libraries](https://nx.dev/structure/library-types), you can extend your project's configuration (i.e., `project.json`) and provide the application's styles. For example: 
```

And update the json snippet to the following:
```json
 "build-storybook": {
    "executor": "@nrwl/storybook:build",
    "outputs": ["{options.outputPath}"],
    "options": {
      "uiFramework": "@storybook/angular",
      "outputPath": "dist/storybook/example-storybook",
      "config": {
        "configFolder": "libs/example-ui-lib/storybook/.storybook"
      },
      "projectBuildConfig": "example-storybook:build-storybook",
      "styles": ["apps/example-app/src/styles.scss"]
    },
    "configurations": {
      "ci": {
        "quiet": true
      }
    }
  }  
```

With this, we turn the text/snippet a bit more agnostic and provide a clearer picture and avoid having what I assume is a working configuration that you've set up for your case.

What do you think?

Reply:
That's great to hear @juri-sinitson ! Let me know once this is addressed and I'll gladly merge it.

Reply:
@jonniebigodes
> Let me know once this is addressed and I'll gladly merge it.

On my opinion this can be merged. 
What do you mean with `addressed` exactly? (Sorry for a possibly very noob question).

Reply:
No need to apologize whatsoever, what I meant to say is to let me know once you've updated the wording and example based on my feedback. Let me know and we'll go from there.

Hope you have a great day!

Reply:
@jonniebigodes 
I've updated it, and I think, it can now be merged.